### PR TITLE
DTSSTCI-744

### PR DESCRIPTION
**Pull request by tomxelliott**
### Change description
Update privacy policy content in update case

NOTE: Welsh content updates to be completed as part of https://tools.hmcts.net/jira/browse/DTSSTCI-745

### JIRA link
https://tools.hmcts.net/jira/browse/DTSSTCI-744

### CODE REVIEW

**Review by pr00279 — APPROVED** [review]

File: src/main/steps/privacy-policy/content.ts [hunk 1]
```diff
@@ -1,167 +1,213 @@
...
+  weUse:
+    "We use Google Analytics to collect data about how a website is used. This anonymous data is shared with Google. Find out about this in our <a class='govuk-link' href='/terms-and-conditions'>terms and conditions</a>.",
+  lawfulBases:
+    'Under the UK General Data Protection Regulation (UK GDPR), the lawful bases we rely on for processing this information are:',
+  consent:
+    'Your consent. You are able to withdraw your consent at any time. You can do this by contacting Disclosure Team, Post point 10.24, 102 Petty France, London, SW1H 9AJ',
+  legalObligation: 'We have a legal obligation.',
+  vitalInterest: 'We have a vital interest.',
+  publicTask: 'We need it to perform a public task.',
+  legitimateInterest: 'We have a legitimate interest',
```
Comment (rajaram-nagarajan) [inline src/main/steps/privacy-policy/content.ts:68]: Minor: Good to have a dot at the end to maintain uniformity.

File: src/main/steps/privacy-policy/content.ts [hunk 2]
```diff
@@ -1,167 +1,213 @@
...
+  weUse:
+    "Rydym yn defnyddio Google Analytics i gasglu data am sut y defnyddir gwefan. Mae'r data cyffredinol hwn yn cael ei rannu â Google. Gallwch gael gwybodaeth am hyn yn ein  <a class='govuk-link' href='/terms-and-conditions'>telerau ac amodau</a>.",
+  lawfulBases:
+    'O dan Reoliad Diogelu Data Cyffredinol y DU (GDPR y DU), y seiliau cyfreithlon yr ydym yn dibynnu arnynt ar gyfer prosesu’r wybodaeth hon yw:',
+  consent:
+    'Eich caniatâd. Gallwch dynnu eich caniatâd yn ôl ar unrhyw adeg. Gallwch wneud hyn drwy gysylltu â’r Tîm Datgelu, Post point 10.24, 102 Petty France, Llundain SW1H 9AJ',
+  legalObligation: 'Mae gennym rwymedigaeth gyfreithiol.',
+  vitalInterest: 'Mae gennym ddiddordeb hanfodol.',
+  publicTask: 'Mae ei angen arnom i gyflawni tasg gyhoeddus.',
+  legitimateInterest: 'Mae gennym ddiddordeb dilys',
```
Comment (rajaram-nagarajan) [inline src/main/steps/privacy-policy/content.ts:173]: Minor: Good to have a dot at the end to maintain uniformity.

**Review by pr00279 — DISMISSED** [review]

File: src/main/steps/privacy-policy/content.ts [hunk 3]
```diff
@@ -1,167 +1,213 @@
...
+    "We use Google Analytics to collect data about how a website is used. This anonymous data is shared with Google. Find out about this in our <a class='govuk-link' href='/terms-and-conditions'>terms and conditions</a>.",
+  lawfulBases:
+    'Under the UK General Data Protection Regulation (UK GDPR), the lawful bases we rely on for processing this information are:',
+  consent:
+    'Your consent. You are able to withdraw your consent at any time. You can do this by contacting Disclosure Team, Post point 10.24, 102 Petty France, London, SW1H 9AJ',
+  legalObligation: 'We have a legal obligation.',
+  vitalInterest: 'We have a vital interest.',
+  publicTask: 'We need it to perform a public task.',
+  legitimateInterest: 'We have a legitimate interest.',
+  storeAndShare: 'Storing and sharing your data internationally.',
```
Comment (rajaram-nagarajan) [inline src/main/steps/privacy-policy/content.ts:69]: storeAndShare shouldn't have dot at the end as this is a heading in bold. Same applies for welsh.

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

**Review by rajaram-nagarajan — APPROVED** [review]

**Pull request merged**